scapy/layers/bluetooth4LE.py

```diff
@@ -11,34 +11,20 @@
 
 from scapy.compat import orb, chb
 from scapy.config import conf
-from scapy.data import MTU, DLT_BLUETOOTH_LE_LL
+from scapy.data import MTU, DLT_BLUETOOTH_LE_LL, DLT_BLUETOOTH_LE_LL_WITH_PHDR
 from scapy.packet import *
 from scapy.fields import *
+from scapy.layers.dot11 import _dbmField
 from scapy.layers.ppi import PPI, addPPIType, PPIGenericFldHdr
 
 from scapy.contrib.ppi_geotag import XLEIntField, XLEShortField
 from scapy.layers.bluetooth import EIR_Hdr, L2CAP_Hdr
 
 from scapy.modules.six.moves import range
 
-
-BTLE_Versions = {
-    7: '4.1'
-}
-BTLE_Corp_IDs = {
-    0xf: 'Broadcom Corporation'
-}
-
-
-class CtrlPDU(Packet):
-    name = "CtrlPDU"
-    fields_desc = [
-        XByteField("optcode", 0),
-        ByteEnumField("version", 0, BTLE_Versions),
-        LEShortEnumField("Company", 0, BTLE_Corp_IDs),
-        XShortField("subversion", 0)
-    ]
-
```

Comment:
Moved to the bottom of the file. `CtrlPDU` sits on `BTLE_DATA` so let's put it after